rmw_connext_dynamic_cpp/src/functions.cpp

```diff
@@ -497,8 +255,6 @@ DDS_TypeCode * _create_type_code(
   return NULL;
 }
 
-extern "C"
-{
```

Comment:
I was suspicious of this initially, but it actually two `extern "C"` sections getting merged.